gocourse02/main.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	zookeeper := NewZookeeper(false, "Mukola")
+	zookeeper.PrintName()
+
+	cage := NewCage(true, 1)
+	fmt.Printf("Cage number %d state =>%t\n", cage.Number, cage.IsBroken)
+	if !zookeeper.IsBusy && cage.IsBroken {
+		zookeeper.FixCage(cage)
+	}
+	fmt.Printf("Cage number %d state =>%t\n", cage.Number, cage.IsBroken)
+
+	var sirko = NewAnimal("Sirko", cage, Cat)
```

Comment:
Для консистенції з `cage :=`, `zookeper :=`
```suggestion
	sirko := NewAnimal("Sirko", cage, Cat)
```

Нижче теж бажано поправити, де `var sirkoSon =` та `var sirkoDaughter =`